wagtail_transfer/static_src/components/ContentImportForm/index.js

```diff
@@ -124,9 +151,13 @@ export default function ImportContentForm({
 
         <li className="transfer numbered">
           <div class="transfer list-container">
-            <h2>Select destination parent page</h2>
+            <h2>
+              {!alreadyExistsAtDestination
+                ? 'Select destination parent page'
+                : 'This page already exists at the destination, and will be updated.'}
+            </h2>
           </div>
-          {sourcePage ? (
+          {sourcePage && !alreadyExistsAtDestination ? (
```

Comment:
Will this hide if `alreadyExistsAtDestination === null`?

Reply:
Do you think it should? Isn't choosing a destination page kind of the 'default' behaviour here? Or should there be some kind of loading state for if it's not yet determined?

Reply:
Hides it now!